test/integration/formatter_test.py

```diff
@@ -0,0 +1,148 @@
+import pytest
+from inspect import cleandoc
+
+
+@pytest.mark.format
+@pytest.mark.parametrize("value,expected", [
+    (None, None),
+    (1, "1"),
+    (1.0, "1.0"),
+])
+def test_no_conversion_specified(connection, value, expected):
+    params = {"value": value}
+    query = "SELECT {value};"
+    actual = connection.execute(query, params).fetchval()
+
+    assert expected == actual
+
+
+@pytest.mark.format
+def test_safe_quote_conversion(connection, schema):
+    params = {"table": (schema, "USERS")}
```

Comment:
here a table with a lower case table name would be good, to check if the quoting actually works

Reply:
This does not work with lower case table name.